node/src/Pos/Wallet/Web/ClientTypes/Types.hs

```diff
@@ -171,36 +181,38 @@ data CAccountMeta = CAccountMeta
 instance Default CAccountMeta where
     def = CAccountMeta "Personal Wallet"
 
--- | Client Account (CA)
--- (Flow type: accountType)
-data CAccount = CAccount
-    { caId        :: !CAccountId
-    , caMeta      :: !CAccountMeta
-    , caAddresses :: ![CAddress]
-    , caAmount    :: !CCoin
-    } deriving (Show, Generic, Typeable)
+----------------------------------------------------------------------------
+-- Wallet structure - requests
+----------------------------------------------------------------------------
 
--- | Query data for account creation
-data CAccountInit = CAccountInit
-    { caInitMeta :: !CAccountMeta
-    , caInitWId  :: !(CId Wal)
-    } deriving (Show, Generic)
+-- | Query data for wallet creation
+data CWalletInit = CWalletInit
```

Comment:
@Martoon-00 we should be careful not to break Daedalus UI. This change won't break it atm because they are using client-api to create this object and make a request. 

But this will be breaking change for exchanges, who are using direct requests to the backend. 

Now, if we want to commit to this change we should add information about api changes to the release document so that other clients can react properly

Reply:
I haven't change any type actually, only reordered them

Reply:
Ok, wanted do double check with you. github actually confused me with diff ordering